Property setters require `libc` package dependency

### 🌍 Environment

 - Your operating system and version: Linux x64
 - Your python version: 3.6.8
 - How did you install python (e.g. apt or pyenv)? Did you use a virtualenv?: no
 - Your rust version (`rustc --version`): rustc 1.35.0-nightly (70f130954 2019-04-16)
 - Are you using the latest pyo3 version? Have you tried using latest master: yes

### 💥 Reproducing

```
use pyo3::prelude::*;

#[pyclass]
struct C {
    #[pyo3(set)]
    a: i32
}

#[pymethods]
impl C {
    #[setter]
    fn b(&self, x: i32) -> PyResult<()> {
        Ok(())
    }
}
```
errors out with:
```
error[E0433]: failed to resolve: use of undeclared type or module `libc`
 --> src/lib.rs:3:1
  |
3 | #[pyclass]
  | ^^^^^^^^^^ use of undeclared type or module `libc`

error[E0433]: failed to resolve: use of undeclared type or module `libc`
 --> src/lib.rs:9:1
  |
9 | #[pymethods]
  | ^^^^^^^^^^^^ use of undeclared type or module `libc`
```
Adding a `libc` dependency in `Cargo.toml` fixes it, obviously.

--- a/pyo3-derive-backend/src/pymethod.rs
+++ b/pyo3-derive-backend/src/pymethod.rs
@@ -182,7 +182,7 @@ fn impl_wrap_init(cls: &syn::Type, name: &syn::Ident, spec: &FnSpec<'_>) -> Toke
         unsafe extern "C" fn __wrap(
             _slf: *mut pyo3::ffi::PyObject,
             _args: *mut pyo3::ffi::PyObject,
-            _kwargs: *mut pyo3::ffi::PyObject) -> libc::c_int
+            _kwargs: *mut pyo3::ffi::PyObject) -> pyo3::libc::c_int
         {
             const _LOCATION: &'static str = concat!(stringify!(#cls),".",stringify!(#name),"()");
             let _pool = pyo3::GILPool::new();
@@ -305,7 +305,7 @@ pub(crate) fn impl_wrap_setter(
         #[allow(unused_mut)]
         unsafe extern "C" fn __wrap(
             _slf: *mut pyo3::ffi::PyObject,
-            _value: *mut pyo3::ffi::PyObject, _: *mut ::std::os::raw::c_void) -> libc::c_int
+            _value: *mut pyo3::ffi::PyObject, _: *mut ::std::os::raw::c_void) -> pyo3::libc::c_int
         {
             const _LOCATION: &'static str = concat!(stringify!(#cls),".",stringify!(#name),"()");
             let _pool = pyo3::GILPool::new();

--- a/src/lib.rs
+++ b/src/lib.rs
@@ -130,12 +130,15 @@ pub use crate::objectprotocol::ObjectProtocol;
 pub use crate::python::{prepare_freethreaded_python, Python};
 pub use crate::type_object::{PyObjectAlloc, PyRawObject, PyTypeInfo};
 
-// We need that reexport for wrap_function
+// Re-exported for wrap_function
 #[doc(hidden)]
 pub use mashup;
-// We need that reexport for pymethods
+// Re-exported for pymethods
 #[doc(hidden)]
 pub use inventory;
+// Re-exported for the `__wrap` functions
+#[doc(hidden)]
+pub use libc;
 
 /// Raw ffi declarations for the c interface of python
 pub mod ffi;

